File: wwwroot/app/components/book/bookings.ts
export class Bookings {
 constructor(
 public ConfirmationNumber: number,
 public Id: number,
 public Username: string,
 public Departure: string,
 public Destination: string,
 public DepartDate: Date, 
 public ArrivalTime: Date,
 public Passengers: number,
 public Price: number
 ) { }
}

File: Models/Bookings.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace FisherAirlines.Models
{
    public class Bookings
    {
        [Key]
        public int ConfirmationNumber { get; set; }

        public string Username { get; set; }

        public int Id { get; set; }

        public string Departure {get; set; }

        public string Destination { get; set; }
        
        public DateTime DepartDate { get; set; }

        [DataType(DataType.Time)]
        [DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "{0:H:mm}")]
        public DateTime ArrivalTime { get; set; }

        public int Passengers { get; set; }

        public int Price { get; set; }

        
         

    }
}